scripts/ci/in_container/_in_container_utils.sh

```diff
@@ -58,9 +58,11 @@ function print_in_container_info() {
 # Cleans up PYC files (in case they come in mounted folders)
 #
 function in_container_cleanup_pyc() {
-    print_in_container_info
-    print_in_container_info "Cleaning up .pyc files"
-    print_in_container_info
+    if [[ ${AIRFLOW_CI_VERBOSE:="false"} == "true" ]]; then
```

Comment:
I feel rather than having to do `${VAR:="default}` for every check we should ensure this variable is set once in the setup/init

Reply:
Ah yeah. IT was a copy&paste victim